apparmor.d/groups/lxqt/lxqt-panel

```diff
@@ -0,0 +1,86 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Alexandre Pujol <alexandre@pujol.io>
+# Copyright (C) 2024 Besanon  <m231009ts@mailfence.com>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/4.0>,
+
+include <tunables/global>
+
+@{exec_path} = @{bin}/lxqt-panel
+profile lxqt-panel @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/app-launcher-user>
+  include <abstractions/audio-client>
+  include <abstractions/dconf-write>
+  include <abstractions/lxqt>
+  include <abstractions/nameservice-strict>
+
+  network inet dgram,
+  network inet6 dgram,
+  network inet stream,
+  network inet6 stream,
+  network netlink raw,
+  network packet dgram,
+
+  @{exec_path} mr,
+
+  @{bin}/exo-open rix,
+  @{bin}/nm-applet rPx,
+  @{bin}/nm-connection-editor rPx,
+  @{bin}/xdg-open rPx,
+  @{bin}/ControlPanel rPx,
+
+  /usr/lib{,32,64}/lxqt-panel/*.so mr, # LXQT-Plugins
+  /usr/lib{,32,64}/lxqt-config/*.so mr, # LXQT-Plugins
+
+  /usr/share/lxqt/helpers/*.desktop r,
+  /usr/share/lxqt/panel/plugins/{,*.desktop} r,
+  /usr/share/desktop-directories/{,**} r,
+  /usr/share/X11/locale/locale.alias r,
+  /usr/share/lxqt/themes/{,**} r,
+
+  /etc/fstab r,
+  /etc/udev/udev.conf r,
+  /etc/machine-id r,
+  /etc/xdg/lxqt-qtxdg.conf r,
+  /etc/xdg/menus/**.menu r,
+  /etc/xdg/menus/applications-merged/ r,
+  /etc/xdg/ui/uistandards.rc r,
+
+  /var/lib/dbus/machine-id r,
+
+  owner @{HOME}/.config/menus/*.menu rw,
+  owner @{HOME}/.config/menus/applications-merged/ r,
+  owner @{HOME}/Desktop/*.desktop rw,
+  owner @{HOME}/Desktop/#@{int} rw,
+  owner @{HOME}/Desktop/*.desktop l -> @{HOME}/Desktop/#@{int},
+  owner @{HOME}/.local/share/desktop-directories/*.directory r,
+  owner @{HOME}/.local/share/gvfs-metadata/{,*} r,
+
+  owner @{user_config_dirs}/lxqt/#* rw,
+  owner @{user_config_dirs}/lxqt/panel.conf rw,
+  owner @{user_config_dirs}/lxqt/panel.conf.lock rwk,
+  owner @{user_config_dirs}/lxqt/panel.conf.@{rand6} rw,
+  owner @{user_config_dirs}/lxqt/panel.conf.@{rand6} l -> @{user_config_dirs}/lxqt/#*,
+  owner @{user_config_dirs}/pulse/{,**} rwk,
+  owner @{user_config_dirs}/ibus/bus/{,**} rw,
+
+  @{run}/udev/data/* r,
```

Comment:
That need to be restricted. Take a look at how it is used in the profiles. You usually only need to copy/paste existing rule from other profiles.